invoice/invoice_test.go

```diff
@@ -129,6 +132,51 @@ func TestOdooInvoiceCreator_CreateInvoiceWithParentID(t *testing.T) {
 		WithInvoiceDate(invoiceDate),
 		WithInvoiceDefaults(invoiceDefaults),
 		WithInvoiceLineDefaults(invoiceLineDefaults),
+		WithInvoiceTitle(invoiceTitle),
+	)
+	require.NoError(t, err)
+}
+
+func TestOdooInvoiceCreator_CreateInvoiceWithCustomTitle(t *testing.T) {
```

Comment:
As it is implemented now i'd not copy the test, as the test before already tests the exact same code path. 

Another option would be table driven tests. Fabian always does them really well: 
https://github.com/appuio/control-api/blob/57168d307ac51edd8b499b5a3b5cd8b77cc74586/apiserver/organization/create_test.go#L26